src/linker/Linker/Annotations.cs

```diff
@@ -244,6 +245,17 @@ public bool IsInstantiated (TypeDefinition type)
 			return marked_instantiated.Contains (type);
 		}
 
+		public void ImplementsStaticInterface (TypeDefinition type)
+		{
+			implements_static_interface_methods.Add (type);
+		}
+
+		public bool IsFromStaticInterface (TypeDefinition type)
```

Comment:
When I first saw this I didn't know what to expect from a method with this name.

Consider naming these `void MarkImplementsStaticInterfaceMethods(TypeDefinition)` and `bool ImplementsStaticInterfaceMethod(TypeDefinition)`.

But I'm not even sure we really need them.